src/themes/teams/components/Text/textVariables.ts

```diff
@@ -29,7 +33,10 @@ export interface TextVariables {
 }
 
 export default (siteVariables): TextVariables => {
+  const colorVariant = '500'
+
   return {
+    colors: mapColorsToScheme(siteVariables, colorVariant),
```

Comment:
With current structure it seems that the change of `color` will not affect `disabled`:

```js
<Text color='red' disabled />
```

And it will have `siteVariables.gray06` as color 😨

Reply:
@layershifter I know, that seems wrong, `disabled` prop should not change color, `disabled` prop should only impact functionality and opacity

@codepretty I can only see you as last author, can you weigh in on this?

Also tagging @notandrew  who implemented the same behavior (`disabled` changes color) for `Button` in #216

Reply:
> should only impact functionality and opacity

Yep, and it seems that it should be a part of a color scheme and will result something like this:
![image](https://user-images.githubusercontent.com/14183168/49931886-87374880-fed0-11e8-9e56-695f344f6ab8.png)

Reply:
@layershifter I understand the sentiment and (maybe?) that is what you would want for some themes, but not for Teams.

The `disabled` prop should dictate color for Teams theme since opacity isn't the correct treatment for disabled states in Teams.

Reply:
@codepretty got it. Thanks for the feedback.